layouts/shortcodes/observability_pipelines/configure_existing_pipelines/source_env_vars/amazon_s3.md

```diff
@@ -0,0 +1,9 @@
+- Amazon S3 SQS URL
+    - The URL of the SQS queue to which the S3 bucket sends the notification events.
+    - Stored as the environment variable: `DD_OP_SOURCE_AWS_S3_SQS_URL`
+- AWS_CONFIG_FILE path
+	- The path to the AWS configuration file local to this node.
```

Comment:
Make indents consistent?

Reply:
Weird, I don't know why it shows up like that. If I try to add a suggestion to fix it, the indent looks correct. 🤔